benchmarks/bench_network_io.py

```
@@ -0,0 +1,237 @@
+    # Start HTTP server on host
+    host_port = _find_free_port()
+    server = _start_http_server(host_port)
+    print(f"  Test HTTP server running on port {host_port}")
+
+    try:
+        ## here we set up a baseline on tyhe host
+        print_subheader("Host: HTTP request to localhost")
+        host_http_times = []
+        for _ in range(100):
+            t = time_call(
+                lambda: subprocess.run(
```

Ruff (0.15.4)

[error] `subprocess` call: check for execution of untrusted input

(S603)